x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/tour_step.tsx

```diff
@@ -11,26 +11,53 @@ import type { EuiTourStepProps } from '@elastic/eui';
 import { EuiButton, EuiImage, EuiSpacer, EuiText, EuiTourStep } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 
+import styled from 'styled-components';
+import { useShallowEqualSelector } from '../../hooks/use_selector';
+import { TimelineId } from '../../../../common/types';
+import { timelineDefaults } from '../../../timelines/store/timeline/defaults';
+import { timelineSelectors } from '../../../timelines/store/timeline';
 import { useTourContext } from './tour';
-import { securityTourConfig, SecurityStepId } from './tour_config';
+import { SecurityStepId, securityTourConfig } from './tour_config';
+
 interface SecurityTourStep {
   children?: React.ReactElement;
   step: number;
   stepId: SecurityStepId;
 }
 
+const isStepExternallyMounted = (stepId: SecurityStepId, step: number) =>
+  step === 5 && stepId === SecurityStepId.alertsCases;
+
+const StyledTourStep = styled(EuiTourStep)<EuiTourStepProps & { stepId: SecurityStepId }>`
+  &.euiPopover__panel[data-popover-open] {
+    z-index: ${({ step, stepId }) =>
+      isStepExternallyMounted(stepId, step) ? '9000 !important' : '1000 !important'};
+  }
+`;
+
 export const SecurityTourStep = ({ children, step, stepId }: SecurityTourStep) => {
   const { activeStep, incrementStep, isTourShown } = useTourContext();
   const tourStep = useMemo(
     () => securityTourConfig[stepId].find((config) => config.step === step),
     [step, stepId]
   );
+
+  const getTimeline = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
+  const showTimeline = useShallowEqualSelector(
```

Comment:
Fixes this

<img width="1288" alt="Screen Shot 2022-10-28 at 10 17 01 AM" src="https://user-images.githubusercontent.com/6935300/198684709-684187e7-6b81-49f7-92d7-069ffdd1fd18.png">